{{cookiecutter.github_repository}}/provisioner/roles/project_data/defaults/main.yml

```diff
@@ -33,4 +35,4 @@ uwsgi_pid_file: "/tmp/uwsgi-{{ project_namespace }}.pid"
 
 uwsgi_log_dir: /var/log/uwsgi
 uwsgi_log_file: "{{ uwsgi_log_dir }}/{{ project_namespace }}.log"
```

Comment:
I think this would need the raw tag, otherwise cookiecutter will try to replace these vars, which I believe are for ansible.